options/options.c

```diff
@@ -52,6 +52,79 @@
 #include "stream/stream.h"
 #include "demux/demux.h"
 
+// Windows specific defines:
+#ifndef DWMWCP_DEFAULT
+#define DWMWCP_DEFAULT 0
+#endif
+
+#ifndef DWMWCP_DONOTROUND
+#define DWMWCP_DONOTROUND 1
+#endif
+
+#ifndef DWMWCP_ROUND
+#define DWMWCP_ROUND 2
+#endif
+
+#ifndef DWMWCP_ROUNDSMALL
+#define DWMWCP_ROUNDSMALL 3
+#endif
+
+#ifndef DWMSBT_AUTO
+#define DWMSBT_AUTO 0
+#endif
+
+#ifndef DWMSBT_NONE
+#define DWMSBT_NONE 1
+#endif
+
+#ifndef DWMSBT_MAINWINDOW
+#define DWMSBT_MAINWINDOW 2
+#endif
+
+#ifndef DWMSBT_TRANSIENTWINDOW
+#define DWMSBT_TRANSIENTWINDOW 3
+#endif
+
+#ifndef DWMSBT_TABBEDWINDOW
+#define DWMSBT_TABBEDWINDOW 4
+#endif
+
+#ifndef WDA_NONE
+#define WDA_NONE 0
+#endif
+
+#ifndef WDA_MONITOR
+#define WDA_MONITOR 1
+#endif
+
+#ifndef WDA_EXCLUDEFROMCAPTURE
+#define WDA_EXCLUDEFROMCAPTURE 0x00000011
+#endif
+
+#ifndef REALTIME_PRIORITY_CLASS
+#define REALTIME_PRIORITY_CLASS 0x00000100
+#endif
+
+#ifndef HIGH_PRIORITY_CLASS
+#define HIGH_PRIORITY_CLASS 0x00000080
+#endif
+
+#ifndef ABOVE_NORMAL_PRIORITY_CLASS
```

Comment:
don't like that we duplicate defines, that would essentially not be used anyway.

Reply:
Wasn't sure about this either. Maybe for non-windows we just generate the defines as bogus 0 values in meson?